### PR TITLE
Pdda

### PR DESCRIPTION
**Pull request by harleyfinleycgi**
### Checklist

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)
- [ ] Does this PR introduce a breaking change

### CODE REVIEW

File: apps/pdda/serviceaccount/demo.yaml [hunk 1]
```diff
@@ -4,4 +4,4 @@ metadata:
   name: ${NAMESPACE}
   namespace: ${NAMESPACE}
   annotations:
-    azure.workload.identity/client-id: "424b1e06-4b81-4d49-a5b4-ed120e6ff736"
+    azure.workload.identity/client-id: ""
```
Comment (thomast1906) [inline apps/pdda/serviceaccount/demo.yaml:7]: Should this be empty?

**Review by thomast1906 — CHANGES_REQUESTED** [review]
Should this be blank?

File: apps/pdda/serviceaccount/demo.yaml (hunk 1 above)
Comment (harleyfinleycgi) [reply apps/pdda/serviceaccount/demo.yaml:7]: yeah we dont have demo, in hindsight i should just delete this

**Review by markharriscgi — APPROVED** [review]

File: apps/pdda/serviceaccount/demo.yaml (hunk 1 above)
Comment (markharriscgi) [reply apps/pdda/serviceaccount/demo.yaml:7]: Reviewed the deletion.

**Review by thomast1906 — APPROVED** [review]

**Pull request merged**